forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRDeviceProfileTest.java

```diff
@@ -99,13 +104,34 @@ public void testDeviceProfile() throws JSONException, ExecutionException, Interr
 
         // Location may not be captured using emulator
         // or if ACCESS_BACKGROUND_LOCATION permission is not granted
-        int backgroundLocationPermissionApproved =
-                ActivityCompat.checkSelfPermission(context,
-                        Manifest.permission.ACCESS_BACKGROUND_LOCATION);
+        boolean locationPermissionGranted = false;
 
-        if (!isEmulator() && backgroundLocationPermissionApproved >= 0) {
-            result.getJSONObject("location").getDouble("latitude");
-            result.getJSONObject("location").getDouble("longitude");
+        // If the location services are NOT enabled skip the rest of the test...
```

Comment:
Attempted to fix random failures on BitBar where for some reason on some divide location data is not available...